responsive-web-design/build-survey-form/desing.css

```diff
@@ -0,0 +1,103 @@
+
+body {
+    background-color: rgb(122, 253, 247);
+    text-align: center;
+    font-family: 'Open Sans', sans-serif;
+    min-width: 320px;
+}
+
+#center {
+    
+    background-color: rgb(255, 255, 255); 
+    border-radius: 6px;
+    border: 1px rgb(5, 5, 5) solid;
+    margin: 0 auto;
+    max-width: 900px;
+    width: 75%;
+    padding: 10px;
+    padding-top: 20px;
+
+}
+
+.input-field {
+    height: 20px;
+    padding: 5px;
+    margin: 10px;
+    width: 280px;
+
+}
+
+@media screen and (max-width: 833px){
+    .input-field {
+        width: 80%;
+    }
+}
+
+
+
+.labels {
+    display: inline-block;
+    text-align: right;
+    width: 40%;
+    padding: 5px;
+    vertical-align: top;
+    margin-top: 10px;
+}
+
+.rightTab{
+    display: inline-block;
+    text-align: left;
+    width: 48%;
+    vertical-align: middle;
+}
+
+
+
+.dropdown{
+    height: 33px;
+    width: 140px;
+    padding: 5px;
+    margin: 10px;
+    margin-left: 10px;
+}
+
+.userRatings, input[type="checkbox"]{
+float: left;
+margin-right: 5px;
+}
+
+.radio, .checkbox{
+    display: block;
+    position: relative;
+    left: -43px;
+    margin-left: 10px;
+    padding-bottom: 10px;
+    
+}
+
+.comment-area{
+    height: 97px;
+    resize: vertical;
+    margin-top: 10px;
+    margin-bottom: 10px;
+}
+
+#submit{
+    background-color: rgb(122, 253, 247);
+    margin: 10px;
+    width: 90px;
+    height: 36px;
+    font-size: 1em;    
+    color: black;
+    border-radius: 3px;
+    border: 1px solid;
+    font-size: 1em;
+}
+
+
+#title {
+    font-weight: 60;
```

Comment:
The value `60` seems to be a not valid value for `font-weight`, if you want to add `bold` format you can use: `font-weight: bold`.

For more information about which values you can use for the `font-weight` please see this page: https://htmldog.com/references/css/properties/font-weight/

Reply:
Yeap, 60 is out of range. Font-weight properties in numbers varies from 100 to 700. I decided to use 100 as it gives the appearance I'm looking for.
Thanks!!!